meteor/client/ui/RundownView.tsx

```diff
@@ -2452,6 +2468,8 @@ export const RundownView = translateWithTracker<IProps, IState, ITrackedProps>((
 										? t('This rundown has been unpublished from Sofie.')
 										: !this.props.studio
 										? t('Error: The studio of this Rundown was not found.')
+										: !this.props.rundowns.length
+										? t('This playlist is empty')
```

Comment:
Since we never refer to playlists in User-facing code, I think this needs to be rephrased? Or we need to use the word *Playlist* more, so that users know what this is. As-is this will just be confusing, and - unlike a missing `showStyleBase` this is something that can easily happen during normal operation.